tests/longevity-tests/Dockerfile.foundation

```diff
@@ -18,12 +18,12 @@ COPY secrets /go/src/github.com/vmware/vic
 ENV LONGEVITY=1
 ENV GOVC_INSECURE=1
 ENV GOVC_USERNAME=administrator@vsphere.local
-ENV GOVC_DATASTORE=vsanDatastore
-ENV TEST_DATASTORE=vsanDatastore
+ENV GOVC_DATASTORE=CPBU_2TB_01
+ENV TEST_DATASTORE= CPBU_2TB_01
 ENV TEST_USERNAME=administrator@vsphere.local
 ENV TEST_TIMEOUT=3m
-ENV TEST_RESOURCE=cls
+ENV TEST_RESOURCE=Cluster
 ENV DOMAIN=
-ENV BRIDGE_NETWORK=bridge
-ENV PUBLIC_NETWORK=vm-network
+ENV BRIDGE_NETWORK=vxw-dvs-3444-virtualwire-1-sid-5000-VCH-LS-1
+ENV PUBLIC_NETWORK=DPortGroup
 ENV DOCKER_API_VERSION=1.23
```

Comment:
Odd, not sure what this does but VIC 1.3.1 supports DOCKER_API_VERSION=1.25. https://vmware.github.io/vic-product/assets/files/html/1.3/vic_app_dev/container_operations.html

Reply:
yea, that has just been set that way for a while. We need a follow up issue to update that, I don't want to try and conflate multiple changes into a single PR.